gprofiler/utils.py

```diff
@@ -97,12 +98,27 @@ def start_process(cmd: Union[str, List[str]], via_staticx: bool, **kwargs) -> Po
             env = env if env is not None else os.environ.copy()
             env.update({"LD_LIBRARY_PATH": ""})
 
+    cur_preexec_fn = kwargs.pop("preexec_fn", os.setpgrp)
```

Comment:
Why the default of `os.setpgrp`?

Reply:
Oh, I see it was used previously. Question for another time then

Reply:
Actually I've seen it today and it was in the Initial commit, I wondered why..